azure-pipelines.yml

```diff
@@ -170,8 +170,7 @@ stages:
           - script: nuget pack nuspec\netcoreglobaltool\dotnet-sonarscanner.nuspec -OutputDirectory build
             displayName: 'Package dotnet global tool'
 
-          - powershell: | # needs to be 'powershell' because wildcard expansion doesn't work with 'script'
-              nuget sign "$env:PACKAGES_PATH" -Overwrite -HashAlgorithm SHA256 -CertificateFingerprint $(SM_CERT_FP) -Timestamper http://timestamp.digicert.com -TimestampHashAlgorithm SHA256
+          - task: PowerShell@2
```

Comment:
Line 170 is not using PowerShell@2 on purpose, because it has no extra setup, right?